selftests/functional/test_thirdparty_bugs.py

```diff
@@ -0,0 +1,29 @@
+import json
+import sys
+from avocado.utils import download
+
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
+
+
+class TestThirdPartyBugs(unittest.TestCase):
+    """
+    Class created to verify third-party known issues
+    """
+
+    def test_paramiko_ecsda_bug(self):
+        # https://github.com/paramiko/paramiko/issues/243
+        # Problems with using ECDSA known_hosts keys when negotiation also
+        # accepts RSA or DSS keys
+        issue_url = 'https://api.github.com/repos/paramiko/paramiko/issues/243'
+        issue = json.load(download.url_open(issue_url))
```

Comment:
Actually not. You need to catch the `URLError` and skip the test when network not available, instead of failure.